[FR/EN] Add `Mapping Techniques/Rhythm`, `Sliders`, `Spinners`, `Unrankable`, improvement of EN files (#5665)
ppy/osu-wiki: wiki/Mapping_Techniques/Rhythm/fr.md

   - Vous pouvez choisir entre le sens des aiguilles d'une montre et le sens inverse, cela n'a pas vraiment d'importance.
5. Cliquez sur `OK`..
6. Voilà, vous avez terminé !

### Espacement accéléré

*À ne pas confondre avec [Spacing Out](#spacing-out).*

Nominateur(s) : [Gabi](https://osu.ppy.sh/users/57057)

L'**espacement accéléré** consiste à espacer de plus en plus chaque objet, tout en conservant le même timing entre eux.

### Battements aller-retour

Nominateur(s) : [kingcobra52](https://osu.ppy.sh/users/9934)

Les **battements aller-retour** sont composés de deux piles de battements (ou plus), avec les notes impaires dans une pile et les notes paires dans l'autre.
Pour l'utiliser, le joueur doit alterner entre les deux piles.

Il est recommandé de ne l'utiliser que pour les difficultés les plus difficiles, car cela peut être difficile pour les nouveaux joueurs.

### Piles différentielles

Nominateur(s) : [NoHItter](https://osu.ppy.sh/users/124455)

Les **piles différentielles** consistent en des notes qui sont espacées de manière incohérente les unes des autres dans la timeline.
Une chose notable est que ces objets sont mappés comme s'ils étaient un battement cohérent.

> Une certaine lecture à haute voix peut être requise, à moins que vous ne puissiez vraiment suivre la musique.

Reviewer: "Sightreading" faudrait surement l'ajouter à la liste des termes parce qu'il est assez répendu est un peu dur à traduire en français. J'ai supprimé la négation parce que globalement le message c'est "Si tu ne sais pas à quoi t'attendre il faut que tu saches vraiment bien suivre la musique"
```suggestion
Du sightreading peut être requis, à moins que vous puissiez vraiment bien suivre la musique.
```

> Lorsque vous jouez une beatmap en utilisant cette technique, il peut être très difficile de jouer avec le modificateur de jeu Hidden.

Reviewer: Ici c'est la beatmap qui utilise la technique
```suggestion
Lorsque vous jouez une beatmap utilisant cette technique, il peut être très difficile de jouer avec le modificateur de jeu Hidden.
```

### Sections de drainage

Nominateur(s) : [SapphireGhost](https://osu.ppy.sh/users/388602)

> Le **sections de drainage** sont des sections dans une beatmap qui deviennent soudainement *plus lentes* ou *moins difficiles* dans le but de mieux correspondre à la chanson ou de faire baisser la barre de vie.

Reviewer: ```suggestion
Les **sections de drainage** sont des sections dans une beatmap qui deviennent soudainement *plus lentes* ou *moins difficiles* dans le but de mieux correspondre à la chanson ou de faire baisser la barre de vie.
```

Cela peut être fait en plaçant moins de notes et/ou en ralentissant la vitesse du slider.

> Les sections de drainage doivent être utilisées avec parcimonie et uniquement lorsque la chanson peut *y faire* allusion.

Reviewer: Je sais pas si ma tournure est la bonne mais je pense qu'il faut changer le "faire allusion"
```suggestion
Les sections de drainage doivent être utilisées avec parcimonie et uniquement lorsque la chanson peut *s'y prêter*.
```

Cela permet de créer une ligne droite.

### Moyeu et rayons

Nominateur(s) : Beuchi-chan

La formation **moyeu et rayons** est littéralement un moyeu avec des rayons qui dépassent.
Cette technique nécessite au moins 8 à 9 objets qui sont séparés d'un demi battement les uns des autres.
Cela oblige le joueur à se déplacer dans une direction circulaire et à revenir au centre après avoir frappé un objet.
En général, il s'agit essentiellement de la technique des [QQ diamonds](/wiki/Mapping_Techniques/Formations#qq-diamonds) avec un centre pour chaque autre objet.

Tout d'abord, placez les combinaisons de numéros impairs au centre de l'endroit où le moyeu et les rayons doivent être mis en correspondance (cette partie est le "moyeu").
Les objets suivants, pairs, doivent ensuite être placés à 0 degré, 90 degrés, 180 degrés, 270 degrés (ou vice versa) autour du "moyeu" (ce sont les "rayons").

Il est très important de disposer les objets de manière à ce qu'ils aillent dans le sens des aiguilles d'une montre ou dans le sens inverse.

La forme de base du moyeu et rayons utilise 4 rayons.

> Vous pouvez toutefois utiliser 3, 5 ou 6 rayons à la place (dans ce cas, veillez à ce que la mesure du degré reste égale pour chaque rayon).

Author: ```suggestion
Vous pouvez toutefois utiliser 3, 5 ou 6 rayons à la place (dans ce cas, veillez à ce que la mesure en degré reste égale pour chaque rayon).
```

> Vous pouvez toutefois utiliser 3, 5 ou 6 rayons à la place (dans ce cas, veillez à ce que la mesure en degré reste égale pour chaque rayon).

Reviewer: Du coup est-ce qu'on dis spoke au lieu de rayon ? xD

Author: On peut laisser comme ça je pense

> ### Aller-retour linéaires

Reviewer: ```suggestion
### Linear Back and Forth
```

> Vous pouvez toutefois utiliser 3, 5 ou 6 rayons à la place (dans ce cas, veillez à ce que la mesure du degré reste égale pour chaque rayon).

Reviewer: J'ai essayé de remplacer degré par centimètre pour voir ce qui était mieux et je trouve que c'est mieux au pluriel même si c'est au singulier en anglais (mais il y a des fois ou des singuliers en anglais se traduisent par des pluriels en français et vice-versa il me semble)
Après je pense qu'il y a une meilleure façon d'écrire cette phrase que celle que je propose x)
```suggestion
Vous pouvez toutefois utiliser 3, 5 ou 6 rayons à la place (dans ce cas, veillez à ce que la mesure des degrés reste égale pour chaque rayon).
```

Nominateur(s) : [Derekku](https://osu.ppy.sh/users/91341)

> L'**aller-retour linéaire** est une variation des [battements aller-retour](#battements-aller-retour).

Reviewer: ```suggestion
Le **linear back and forth** est une variation des [back and forth beats](#back-and-forth-beats ).
```

> L'**aller-retour linéaires** est une variation des [battements aller-retour](#battements-aller-retour).

Reviewer: ```suggestion
L'**aller-retour linéaire** est une variation des [battements aller-retour](#battements-aller-retour).
```

La différence réside dans le fait que ces derniers n'utilisent pas de piles, mais demandent au joueur de se déplacer entre les objets.

> Il est très important que le mappeur indique clairement la direction dans laquelle se dirige la propagation.

Reviewer: Je pense qu'il faut trouver autre chose que "propagation"
```suggestion
Il est très important que le mappeur indique clairement la direction dans laquelle se dirige l'espacement.
```

### Spacing Out

*À ne pas confondre avec l'[Espacement accéléré](#espacement-accéléré)*

Nominateur(s) : [Pasonia](https://osu.ppy.sh/users/43345)

> Le **spacing out** fait référence aux objets dans lesquels ils sont placés en termes de distance, et non de temps.

Reviewer: ```suggestion
Le **spacing out** fait référence aux objets tels qu'ils sont placés en termes de distance, et non de temps.
```

Cette technique a l'avantage de maintenir l'élan d'un morceau pendant cette section de la map.

> Cependant, un patternmapping incorrect peut conduire à une map presque injouable.

Reviewer: Je chipote un peu là j'avoue, je trouvais juste que ça faisait juste un peu plus fluide comme ça donc à toi de voir :^)
```suggestion
Cependant, un patternmapping incorrect peut conduire à une map quasiment injouable.
```

En gros, formez un carré autour du centre avec 4 cercles.
Les 4 cercles suivants doivent ensuite être placés 1 espace de grille plus loin et ainsi de suite.

Une autre variante que vous pouvez ajouter est de placer un cercle au centre après chaque cycle.

### Step-Too-Fars

Nominateur(s) : [OzzyOzrock](https://osu.ppy.sh/users/465153)

> Le **step-too-fars** utilise une trajectoire linéaire de cercles qui sont espacés d'un quart de temps.

Reviewer: Je pense que c'est au pluriel bien que ce ne soit pas clair sur le wiki anglais
```suggestion
Les **step-too-fars** utilisent une trajectoire linéaire de cercles qui sont espacés d'un quart de temps.
```

Cependant, lorsqu'ils atteignent la "fin" de la ligne, les objets reculent d'une case (le cercle avant la fin de la ligne) puis changent de direction.

## Vitesse

> ### Ralentissements des sliders

Reviewer: ```suggestion
### Slider Slowdowns
```

Nominateur(s) : [123MegaApple123](https://osu.ppy.sh/users/2148208)

> Les **ralentissements de sliders** ralentissent brusquement un slider en divisant temporairement par deux un point de timing hérité (les verts) pour compléter la note longue de la musique (ou de la voix).

Reviewer: ```suggestion
Les **slider slowdowns** ralentissent brusquement un slider en divisant temporairement par deux un point de timing hérité (les verts) pour compléter la note longue de la musique (ou de la voix).
```

## Streams

*Voir [Techniques de mapping/Basique](/wiki/Mapping_Techniques/Basics#streams) pour une description des streams.*

> ### Streams compressés

Reviewer: ```suggestion
### Compressed Streams
```

Nominateur(s) : [pieguyn](https://osu.ppy.sh/users/107485)

> Les **streams compressés** utilisent des cercles qui ne sont séparés que par 3 espaces de grille (sur un niveau à 2 ou 3 grilles).

Reviewer: ```suggestion
Les **compressed streams** utilisent des cercles qui ne sont séparés que par 3 espaces de grille (sur un niveau à 2 ou 3 grilles).
```

Sinon, il suffit de rendre le stream considérablement plus petit par rapport au reste de la map.

La majorité des maps qui utilisent des streams sont plus susceptibles d'utiliser des streams compressés.

> ### Streams discontinus

Reviewer: ```suggestion
### Discontinuous Streams
```

> La majorité des maps qui utilisent des streams sont les plus susceptibles d'être vues en utilisant des steams compressés.

Reviewer: Je suis pas sûr du sens de "To be seen" (peut-être "en fait" dans ce cas là ça donnerait "La majorité des maps qui utilisent des streams sont en fait plus susceptibles d'utiliser des streams compressés.") dans la traduction originale mais je pense qu'on a pas trop besoin de s'en soucier et je pense que ça ne change quasiment rien au sens de la phrase en l'ôtant. Désolé pour le commentaire un peu chargé xD
```suggestion
La majorité des maps qui utilisent des streams sont plus susceptibles d'utiliser des streams compressés.
```

Nominateur(s) : [benguin](https://osu.ppy.sh/users/1322807)

> On parle de **streams discontinus** lorsqu'il y a un *jump* dans le stream, même si le temps entre les notes du jump est de la même longueur que le temps entre les notes du reste du stream.

Reviewer: ```suggestion
On parle de **discontinuous streams** lorsqu'il y a un *jump* dans le stream, même si le temps entre les notes du jump est de la même longueur que le temps entre les notes du reste du stream.
```

> On parle de **streams discontinu** lorsqu'il y a un *jump* dans le stream, même si le temps entre les notes du jump est de la même longueur que le temps entre les notes du reste du steam.

Reviewer: ```suggestion
On parle de **streams discontinus** lorsqu'il y a un *jump* dans le stream, même si le temps entre les notes du jump est de la même longueur que le temps entre les notes du reste du stream.
```

### Deathstreams

Nominateur(s) : [NoHItter](https://osu.ppy.sh/users/124455)

Les **deathstream** sont *très* longs et difficiles.
Il s'agit de sliders (avec ou sans répétitions), de cercles ou d'une combinaison des deux.

> Le travail d'un deathstream est de faire échouer pratiquement les joueurs qui ne peuvent pas gérer les streams ordinaires.

Reviewer: ```suggestion
Le but d'un deathstream est de pratiquement faire échouer les joueurs qui ne peuvent pas gérer les streams ordinaires.
```

L'utilisation de deathstreams rendra votre map **unrankable**!
Cependant, il peut être accepté comme l'une des beatmaps *approuvée*.

### Slider Stream Jumps

Nominateur(s) : [dkun](https://osu.ppy.sh/users/154400) et [MMzz](https://osu.ppy.sh/users/128993)

Le **slider stream jumps** est une technique plus avancée pour osu!.

> Au lieu d'utiliser un stream orthodoxe, vous le remplacez par des sliders.

Reviewer: ```suggestion
Au lieu d'utiliser un stream ordinaire, vous le remplacez par des sliders.
```

Cela permet de conserver la même difficulté de stream, car vous devez vous déplacer et frapper plutôt que de simplement suivre un stream.

Essentiellement, vous sautez avec votre slider et votre stream, d'où le terme *Slider Stream Jumps*.

### Triplet Streams

Nominateur(s) : [Lybydose](https://osu.ppy.sh/users/64501)

> Un **triplet streams** utilise un ensemble de trois cercles qui doivent être frappés en succession rapide.

Reviewer: Pareil que pour les "step-too-fars"
```suggestion
Les **triplet streams** utilisent un ensemble de trois cercles qui doivent être frappés en succession rapide.
```

Ceux-ci sont généralement placés dans un diviseur d'un quart de temps en fonction du BPM de la map.

> Ils peuvent l'être en une seule succession ou en plusieurs, selon ce que souhaite le mappeur.

Reviewer: ```suggestion
Ils peuvent être en une seule succession ou en plusieurs, selon ce que souhaite le mappeur.
```

Notez que Triplet Steams ne fait **pas** référence à un triplet réel en [théorie musicale](/wiki/Beatmap_Editor_Guides/Music_Theory).

> ### Streams à espacement variable

Reviewer: ```suggestion
### Variable Spaced Streams
```

Nominateur(s) : [fartownik](https://osu.ppy.sh/users/56917)

> Les **streams à espacement variable** utilisent des streams plus longs dont l'espacement est irrégulier (généralement croissant).

Reviewer: ```suggestion
Les **variable spaced streams** utilisent des streams plus longs dont l'espacement est irrégulier (généralement croissant).
```

> ### Streams en zig-zag

Reviewer: ```suggestion
### Zig-Zag Streams
```

Nominateur(s) : [Daru](https://osu.ppy.sh/users/32480)

> Les **streams en zig-zag** sont des streams dont les objets sont placés de telle manière qu'ils forment une ligne très dentelée.

Reviewer: ```suggestion
Les **zig-zag streams** sont des streams dont les objets sont placés de telle manière qu'ils forment une ligne très dentelée.
```
